scripts/build-color-variants.js

```diff
@@ -480,11 +480,13 @@ function buildColorVariants(variables, config) {
       const colorValue = variables[color];
       return result += stripIndent(`
         .shadow-${color}-on-hover:hover,
-        .shadow-${color}-on-active.is-active {
+        .shadow-${color}-on-active.is-active,
+        .shadow-${color}-on-active.is-active:hover {
```

Comment:
`.x:hover` and `.x.y` have the same specificity. So whichever comes later in the stylesheet should override. You then don't need to create rules for `.x.y:hover`.